### DISCUSSION TITLE
Milvus uri

### DISCUSSION THREAD

**Discussion by flefevre** [open-webui/open-webui#6170] [Q&A]
In the documentation, it seems we could use milvus standalone vector database
Could you provide an example of the Uri parameters?
Https:// or http?
How to code the port?
Thanks for your advice

MILVUS_URI
Type: str
Default: ${DATA_DIR}/vector_db/milvus.db
Description: Specifies the URI for connecting to the Milvus vector database. This can point to a local or remote Milvus server based on the deployment configuration.

**Comment by Mgrsc** [comment]
How do I set tokens in the Milvus example on the Zilliz website?

**Comment by forcewake** [comment]
expected form `http[s]://[user:password@]example.com[:12345]`

**Comment by santosh0787** [comment]
Hello, has anyone discovered a way to pass a username/password or token for Milvus(Zilliz) DB?

**Reply by ykiran** [reply]
The default creds are `root:Milvus.`
If you're running locally and default config, you can use it as: http://root:Milvus@localhost:19530

It'll connect, but will throw an error regarding missing path (which is the default path specified).
Currently, there seems to be no way to specify the database name.
[Relevant Discussion](https://github.com/open-webui/open-webui/discussions/7142)

**Comment by flefevre** [comment]
finally we are using in our values milvus-uri: http://owui_user:<owui_secret>@<milvus-svc>-<milvus-namespace>:19530
